lib/internal/Magento/Framework/Locale/Test/Unit/FormatTest.php

```diff
@@ -82,7 +82,7 @@ public function getPriceFormatDataProvider()
         return [
             ['en_US', ['decimalSymbol' => '.', 'groupSymbol' => ',']],
             ['de_DE', ['decimalSymbol' => ',', 'groupSymbol' => '.']],
-            ['de_CH', ['decimalSymbol' => '.', 'groupSymbol' => '\'']],
```

Comment:
This needs to be investigated further.

This case is failing on my local machine as well but it is passing on Bamboo and Travis CI.

Please rework a test in such a way that it passes both on Travis CI https://travis-ci.org/magento/magento2/jobs/342788625 and your local machine.